src/test-reader/test-parser.js

```diff
@@ -15,11 +15,13 @@ const clearRequire = require("clear-require");
 const path = require("path");
 
 class TestParser extends EventEmitter {
+    #opts;
     #buildInstructions;
 
-    constructor() {
+    constructor(opts = {}) {
```

Comment:
I strongly suggest having explicit boolean option with a sensible name, like "removeJsImports" (if I got right what it does)

Now, when we pass testRunEnv, and get different imports behavior, it looks very unobvious and every time someone will use this class one'd need to investigate how opts are used to understand what behaviour will change.

Also, JSDoc comment with explicit opts type declaration would be really nice to have.